zeppelin-web/src/app/jobmanager/jobmanager.filter.js

```diff
@@ -38,6 +39,17 @@
         });
       }
 
+      if (isSortByAsc === true) {
+        filterItems = _.sortBy(filterItems, function(sortItem) {
+          return sortItem.noteName;
+        });
+      } else {
+        filterItems = _.sortBy(filterItems, function(sortItem) {
```

Comment:
@cloverhearts  Thanks for you check.
And let's consider upgrading lodash v4 later.
LGTM.